### ISSUE TITLE
Minor issue in readme with quotes

### ISSUE THREAD

**Issue by richardtape** [Alle-AI/anthropic-api-php#3]
In the readme you have the following, as part of an example:

```php
'prompt' => '\n\nHuman: '.$prompt.'\n\nAssistant:', // Be sure to format prompt appropriately
```

However, the `\n\n` string in PHP needs to be in double quotes for it to be parsed properly across all versions of PHP. So that line needs to be 

```php
'prompt' => "\n\nHuman: " . $prompt . "\n\nAssistant:", // Be sure to format prompt appropriately
```

It sounds minor, but it actually prevents the example working if it's copied precisely. (making your inline comment very accurate ;) )

again, I'm happy to submit a Pull Request for this should you wish.

**Comment by dicksonagyei** [comment]
great. Thanks a lot!

**Comment by dicksonagyei** [comment]
Good catch from 2023 — the entire README has been rewritten for v2.0.0 and now uses the structured Messages API (`messages: [['role' => 'user', 'content' => 'Hello']]`), so the single-quote escape problem can't recur. The `Human:` / `Assistant:` prompt format is gone everywhere in the v2 surface.

Closing as obsolete. Thanks for taking the time to file it back then — that kind of attention to detail matters.
